CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [12.0.0-alpha.1](https://github.com/chanzuckerberg/edu-design-system/compare/v12.0.0-alpha.0...v12.0.0-alpha.1) (2023-04-04)
+
```

Comment:
Oop alpha publish changelogs snuck in here but shouldn't matter

Reply:
I looked back and saw a few other older builds included alpha stuff in the change log. This shouldn't interfere with any releases, but makes the change log slightly weird